kupyna/README.md

```diff
@@ -0,0 +1,85 @@
+# Kupyna Hash Function Implementation in Rust
+
+## Overview
+
+The Kupyna hash function is a cryptographic hash developed in Ukraine, designed for high security and efficiency. This implementation supports generating hash codes of various lengths.
+
```

Comment:
Please use other hash crate README as a template, e.g. see the [`streebog`'s README](https://github.com/RustCrypto/hashes/blob/master/streebog/README.md).